test/widgets/app_test.dart

```diff
@@ -51,4 +54,56 @@ void main() {
       ]);
     });
   });
+
+  group('choose account page', () {
+    Future<void> setupChooseAccountPage(WidgetTester tester, {
+      List<Account>? accounts,
+    }) async {
+      addTearDown(testBinding.reset);
+
+      for (final account in accounts ?? []) {
+        await testBinding.globalStore
+          .insertAccount(account.toCompanion(false));
+      }
+
+      await tester.pumpWidget(
+        const MaterialApp(
+          localizationsDelegates: ZulipLocalizations.localizationsDelegates,
+          supportedLocales: ZulipLocalizations.supportedLocales,
+          home: GlobalStoreWidget(
+            child: ChooseAccountPage())));
+
+      // global store gets loaded
+      await tester.pumpAndSettle();
+    }
+
+    List<Account> dummyAccounts(int count) {
```

Comment:
```suggestion
    List<Account> generateAccounts(int count) {
```

Better to avoid "dummy"; it has another meaning as an insult (literally "person who can't talk", commonly used for "stupid person").